packages/react-native/src/primitives/Button/styles.ts

```diff
@@ -1,6 +1,23 @@
-import { StyleSheet } from 'react-native';
+import { Theme } from '../../theme';
 import { ButtonStyles } from './types';
 
-export const styles: ButtonStyles = StyleSheet.create({
+export const baseStyles: ButtonStyles = {
+  borderWidth: 1,
+  borderRadius: 4,
+  padding: 8,
   text: { alignSelf: 'center' },
-});
+};
+
+export const getThemedStyles = (theme: Theme): ButtonStyles => {
+  const buttonToken = theme.tokens?.components.button;
+  return {
+    borderColor:
+      buttonToken?.borderColor ??
+      theme.tokens?.colors.brand?.primary[100].value,
```

Comment:
Slightly concerned about this pattern, why do we need all of the optional chaining and nullish coalescing if the `theme` is a defined structure?

Reply:
added loose theme type